packages/sanity/src/core/studio/components/navbar/search/definitions/operators/arrayOperators.ts

```diff
@@ -0,0 +1,142 @@
+import {type ReferenceValue} from '@sanity/types'
+import {
+  SearchButtonValueNumberCount,
+  SearchButtonValueNumberCountRange,
+  SearchButtonValueReference,
+  SearchButtonValueString,
+} from '../../components/filters/common/ButtonValue'
+import {SearchFilterNumberInput} from '../../components/filters/filter/inputs/number/Number'
+import {SearchFilterNumberRangeInput} from '../../components/filters/filter/inputs/number/NumberRange'
+import {SearchFilterReferenceInput} from '../../components/filters/filter/inputs/reference/Reference'
+import {SearchFilterStringListInput} from '../../components/filters/filter/inputs/string/StringList'
+import {GteIcon} from '../../components/filters/icons/GteIcon'
+import {GtIcon} from '../../components/filters/icons/GtIcon'
+import {LteIcon} from '../../components/filters/icons/LteIcon'
+import {LtIcon} from '../../components/filters/icons/LtIcon'
+import {OperatorNumberRangeValue} from './common'
+import {defineSearchOperator, SearchOperatorButtonValue, SearchOperatorInput} from './operatorTypes'
+import {toJSON} from './operatorUtils'
+
+// @todo: don't manually cast `buttonValueComponent` and `inputComponent` once
```

Comment:
@snorrees A lot of manual casting has been added to these operators, purely to unblock the failing `etl` GH action since `api-extractor` wasn't able to follow these symbols for some reason. Would love your eye on this on this in the near-future